src/Symfony/Component/DependencyInjection/Tests/Dumper/PhpDumperTest.php

```diff
@@ -772,7 +754,7 @@ public function testNonSharedLazy()
         $container = new ContainerBuilder();
 
         $container
-            ->register('foo', Foo::class)
+            ->register('foo', \Bar\FooLazyClass::class)
```

Comment:
A minor mistake I made in my original PR, doesn't really change anything, but it makes the test a bit more consistent.